src/main/scala/fs2/ftp/UnsecureFtp.scala

```diff
@@ -103,6 +103,14 @@ object UnsecureFtp {
                 ftpClient.setConnectTimeout(settings.connectTimeOut)
                 ftpClient.setDefaultTimeout(settings.timeOut)
 
+                //https://enterprisedt.com/products/edtftpjssl/doc/manual/html/ftpscommands.html
+                ftpClient match {
+                  case client: JFTPSClient =>
+                    client.execPBSZ(0)
+                    client.execPROT("P")
+                  case _ => ()
+                }
```

Comment:
Thanks again for your investigation and your PR @jacklockyerstevens 

you could do something like this

```scala
val ftpClient = if (settings.ssl) new JFTPSClient() else {
   val c = new JFTPClient()
   c.execPBSZ(0)
   c.execPROT("P")
   c
}